lib/ViewModels/SettingsPanelViewModel.js

```diff
@@ -22,6 +22,15 @@ var SettingsPanelViewModel = function(options) {
     this.baseMaps = defaultValue(options.baseMaps, []).slice();
     this.mouseOverBaseMap = undefined;
 
+    var that = this;
+    if (defined(options.baseMapName)) {
+        this.baseMaps.map (function(item) {
+            if (options.baseMapName === item.catalogItem.name) {
+                that.terria.baseMap = item.catalogItem;
```

Comment:
This needs to go somewhere else, because the base map specification needs to work even in an application that doesn't use the `SettingsPanelViewModel`.  I'll see if I can find a better place.